lib/cleaned_output.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+require 'colorize'
+
+class CleanedOutput
+  def initialize(output:, morph_api_key: ENV['MORPH_API_KEY'])
+    @output = output
+    @morph_api_key = morph_api_key
+  end
+
+  def to_s
+    (cleaned_output[-64_000..-1] || cleaned_output).uncolorize
```

Comment:
The magic number here should probably be hoisted out. (We'd probably want to take the size as an optional constructor argument too, but I'm OK with leaving that part to a later enhancement if it's not as trivial as it seems it should be)